src/AlgebraicGeometry/Surfaces/EnriquesAut.jl

```diff
@@ -0,0 +1,944 @@
+  [2^5 * 3^2,"96A", [(:A, 1) for i in 1:8]],
+  [2^3 * 3^2,"96B", [(:A, 1) for i in 1:16]],
+  [2^3 * 3^2,"96C", [(:A, 1) for i in 1:4]]]
+  for i in d
+    sort!(i[3])
+  end
+  return d
+end
+
+# Let K be the kernel of O(L26,SX,SY)->O(SX). G^+ is defined as the image of K in O(SY).
+function _compute_Gplus(SY::ZZLat, SX::ZZLat, L26::ZZLat)
+
+  # SY + Sm < SX is a primitive extension with glue map phi: D(Sm) -> D(SY)
+  Sm = orthogonal_submodule(SX, SY)
+  phi, inc_Dminus, inc_Dplus = glue_map(SX, Sm, SY)
+
+  # Cook up the membership test
+  @vprint :EnriquesAuto 2 "computing orthogonal group\n"
+  OSm = orthogonal_group(Sm)
+  GSm,_= image_in_Oq(Sm)
+  phiSm = hom(OSm, GSm, GSm.(gens(OSm)); check=false)
+  DSm = discriminant_group(Sm)
+  Dminus = domain(phi)
+  @vprint :EnriquesAuto 2 "computing stabilizer "
+  stab_Dminus, inc_stab = stabilizer(GSm, inc_Dminus)
+  @vprint :EnriquesAuto 2 "done\n"
+  Dminus_perp, inc_Dminus_perp = orthogonal_submodule(DSm, Dminus)
+  res, inc = restrict_automorphism_group(stab_Dminus, inc_Dminus_perp)
+  res2, inc2 = restrict_automorphism_group(stab_Dminus, inc_Dminus)
+  @vprint :EnriquesAuto 2 "computing kernel "
+  Gminus, inc_Gminus = kernel(inc)
+  @vprintln :EnriquesAuto 2 "done"
+
+  # compute Gplus
+  Q = orthogonal_submodule(L26,SY)
+  DQ = discriminant_group(Q)
+  glue_SY_Q,inc1,inc2 = glue_map(L26,SY, Q)
+  DO = codomain(glue_SY_Q)
+
+  @vprint :EnriquesAuto 2 "preparing membership test\n"
+  DSY = discriminant_group(SY)
+  ODSY = orthogonal_group(DSY)
+  tmp = [preimage(phiSm, i) for i in small_generating_set(Gminus)]
+  tmp_hom = [hom(DO,DO, [DO(lift(j)*i) for j in gens(DO)]) for i in tmp]
+  gens_Gplus = elem_type(ODSY)[ODSY(inv(inc1)*glue_SY_Q*i*inv(glue_SY_Q)*inc1) for i in tmp_hom]
+  Gplus,_ = sub(ODSY, gens_Gplus)
+  @assert order(Gplus) == order(Gminus)
+  # iso_Gplus_minus = hom(Gplus, Gminus, gens(Gplus), small_generating_set(Gminus))
+  return Gplus
+end
+
+
+@doc raw"""
+    isomorphism_classes_elliptic_fibrations(Y::EnriquesBorcherdsCtx)
+
+Return the list of isomorphism classes of elliptic fibrations on the Enriques surface ``Y``.
+Let 
+```math 
+\mathcal{P}^e \colon \mathcal{M}_{En,e} \to \mathcal{M}_{En}, \quad  (Y,f) \mapsto Y
+```
+be the forgetful map from the moduli space of elliptic Enriques surfaces to the moduli space of Enriques surfaces. Here ``f`` is the numerical class of a half-fiber.
+
+# Output:
+A list of triples with the following entries:
+- the ramification index of ``\mathcal{P}^e`` in ``(V,f)``
+- the simple fibers, the double fibers
+- representative of the half fiber ``f`` modulo ``2``
+"""
+function isomorphism_classes_elliptic_fibrations(Y::EnriquesBorcherdsCtx)
+  return  [(length(fbar), reducible_fibers(Y, representative(fbar)),representative(fbar)) for fbar in _ellfib_reps(Y.SY, Y.SX, Y.L26)]
+end 
+  
+function _ellfib_reps(SY::ZZLat, SX::ZZLat, L26::ZZLat)
+  Gplus = _compute_Gplus(SY,SX,L26)
```

Comment:
```suggestion
  Gplus = _compute_Gplus(SY, SX, L26)
```
Is not `Gplus` already computed for `Y::EnriquesBorcherdsCtx` ?